### ISSUE TITLE
can not find the object crop in result 

### ISSUE THREAD

**Issue by Raymond419** [jfzhang95/pytorch-deeplab-xception#192]
Hi all:
Thanks jaemin93 provide inference.py. I used it,but i met some problem that was i can not get the crop of my object.This is the training parameter: Acc:0.9520538766597383,Acc_class:0.9498111995392555, mIoU:0.8039776748270511, fwIoU: 0.9188728055057271. But I could not catch my object target in crop result. All the photo was just white(background).My purpose is to get the black particle in the background.Could someone give some advices to me or tell me something problem in my process

![Screenshot from 2020-12-13 17-18-42](https://user-images.githubusercontent.com/56592375/102009185-ba876900-3d70-11eb-9dcc-4101b47a3925.png)

![363](https://user-images.githubusercontent.com/56592375/102009200-d4c14700-3d70-11eb-88a2-1170a0136496.jpg)

**Comment by Ellsionjeep** [comment]
Are you using your own data set? Miou is pretty high. The result of my own training on voc 2012 is only about 65%. Have you trained on it?

**Comment by Raymond419** [comment]
Yes, I trained my own dataset. Would u know my problem? I think maybe the model recognize the particle is not well. But I do not know excatly real problem is.
